app/screens/reschedule_draft/reschedule_draft.tsx

```diff
@@ -104,14 +108,17 @@ const RescheduledDraft: React.FC<Props> = ({
 
     const onSavePostMessage = useCallback(async () => {
         setIsUpdating(true);
-        setErrorLine(undefined);
-        setErrorExtra(undefined);
         toggleSaveButton(false);
         if (!selectedTime.current) {
             logDebug('ScheduledPostOptions', 'No time selected');
             setIsUpdating(false);
             const errorMessage = intl.formatMessage({id: 'mobile.scheduled_post.error', defaultMessage: 'No time selected'});
-            setErrorLine(errorMessage);
+            showSnackBar({
+                barType: SNACK_BAR_TYPE.RESCHEDULED_POST,
+                customMessage: errorMessage,
+                keepOpen: true,
```

Comment:
I keep on seeing this and wondering the reason

Reply:
Removing it.